index.bs

```diff
@@ -1776,6 +1777,29 @@ This [=internal method=] accepts no arguments.
 
 </div>
 
+### Availability of [=User-Verifying Platform Authenticator=] (enum <dfn enum>UserVerifyingPlatformAuthenticatorAvailability</dfn>) ### {#enum-userVerifyingPlatformAuthenticatorAvailability}
+
+<xmp class="idl">
+    enum UserVerifyingPlatformAuthenticatorAvailability {
+        "enabled",
+        "disabled",
+        "unavailable"
+    };
+</xmp>
```

Comment:
This differentiation of whether the platform authenticator is configured is one bit of information that's not determined by the User-Agent header. On the other hand, I guess the same conclusions can probably be drawn already from a combination of User-Agent and `isUserVerifyingPlatformAuthenticatorAvailable()` result? (i.e., this leaks fingerprinting information, but no more than we already do?)

Reply:
This is for the scenario where RP can actually promote Platform authenticator if not configured. It is the same info from  user-agent and `isUserVerifyingPlatformAuthenticatorAvailable`. We just want to get away from user-agent handling all together. That is too fragile.

Reply:
Sorry, I didn't mean to say User-Agent matching is preferable, just pondering whether this would leak additional fingerprinting information. I think it won't since we already have `isUVPAA()`, but I figured I'd think aloud a bit in case someone else would notice something I didn't.

Reply:
UVPAA modifications have been removed as we don't have a concrete use case for this today.

Reply:
We have concrete use cases for this. RP tending to leverage the platform authenticator and promote it to users has no means to promote the user if they do not turn on the feature even the platform has the function.
For example, we would like to show following information.
1. If the user has platform authenticator and is available: "Try now"
2. If the user has platform authenticator and is disabled: "Your device has the best option for protecting your account. For trying this, please go to setup and turn on the feature. Then, try again"
3. If the user has no platform authenticator, never show any authentication promotion

We have seen many of people haven't setup Windows Hello feature on their device.